static/js/components/EmploymentTab.js

```diff
@@ -12,10 +12,14 @@ import moment from 'moment';
 
 import { generateNewWorkHistory } from '../util/util';
 import ProfileTab from "../util/ProfileTab";
-import { saveAndContinue } from '../util/profile_edit';
+import { saveProfileStep } from '../util/profile_edit';
 
 class EmploymentTab extends ProfileTab {
-  nextUrl = "/profile/privacy";
+  constructor(props) {
+    super(props);
+    this.prevUrl = "/profile/education";
+    this.nextUrl = "/profile/privacy";
```

Comment:
i do too, but overriding those in child classes causes an error (or maybe just a warning?)

eg: PrivacyTab sets nextButtonText to "I'm Done!" instead of the default "Save And Continue". if i set that variable as you suggest and override it in PrivacyTab, i get the error/warning